src/ModuleList/Values/Values.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import styled from '@emotion/styled';
+
+interface Params {
+  valuesId: string;
+};
+
+const getMockData = (valuesId: string) => valuesId === '5001'
```

Comment:
Can you define a type for the Values? It should look something like below... that will help in the future when we pull the values from the server.
```ts
interface Value {
  value: string;
  inspiration: {
    organization: string;
    definition: string;
  }[];
}
```